scss/_variables.scss

```diff
@@ -646,4 +646,4 @@ $kbd-bg:                      #333 !default;
 $pre-bg:                      #f7f7f9 !default;
 $pre-color:                   $gray-dark !default;
 $pre-border-color:            #ccc !default;
-$pre-scrollable-max-height:   340px !default;
+$pre-scrollable-max-height:   340px !default;
```

Comment:
Thanks, I fixed it.
*(Sorry, I deleted previous comment because I decided that @houndci is a bot and will not be able to answer me.)*